package.json

```diff
@@ -28,23 +28,24 @@
   ],
   "dependencies": {
     "aws-sign2": "~0.7.0",
-    "aws4": "^1.8.0",
+    "aws4": "^1.12.0",
     "caseless": "~0.12.0",
-    "combined-stream": "~1.0.6",
+    "combined-stream": "~1.0.8",
     "extend": "~3.0.2",
     "forever-agent": "~0.6.1",
-    "form-data": "~2.3.2",
+    "form-data": "~4.0.0",
     "http-signature": "~1.3.6",
     "is-typedarray": "~1.0.0",
     "isstream": "~0.1.2",
     "json-stringify-safe": "~5.0.1",
-    "mime-types": "~2.1.19",
+    "mime-types": "~2.1.35",
     "performance-now": "^2.1.0",
-    "qs": "~6.10.3",
-    "safe-buffer": "^5.1.2",
-    "tough-cookie": "~2.5.0",
+    "qs": "~6.11.2",
+    "safe-buffer": "^5.2.1",
+    "tough-cookie": "~4.1.3",
     "tunnel-agent": "^0.6.0",
-    "uuid": "^8.3.2"
+    "uuid": "^9.0.0",
+    "watchify": "^4.0.0"
```

Comment:
@legobeat

Good points!

```
"engines": {
    "node": ">= 6"
  },
```

and [uuid 9.0.0](https://github.com/uuidjs/uuid/blob/main/CHANGELOG.md#900-2022-09-05) says it drops Node.js 10.x support.

So this would need to be a breaking change with a new major version.

Given that the PR fails CI tests, in its current form it is not a serious candidate for merging.

Reply:
> move out breaking changes (possibly filing those separately after

Can't quite follow you